src/main/java/coms/w4156/moviewishlist/controllers/WishlistController.java

```diff
@@ -26,78 +28,57 @@ public class WishlistController {
     @Autowired
     WishlistService wlService;
 
-    @GetMapping("/")
+    @Autowired
+    UserService userService;
+
+    @GetMapping
```

Comment:
We don't want to require a trailing `/`

Previously, only `/wishlists/` would work. Now `/wishlists` works too.